packages/itwinui-css/backstop/tests/table.html

```diff
@@ -712,7 +701,7 @@ <h2>Default</h2>
               <div class="iui-table-cell-start-icon">
                 <svg-filetype-vector aria-hidden="true"></svg-filetype-vector>
               </div>
-              MyFileWithAReallyLongNameThatWillBeSplitIntoMultipleLinesAfterSoLong_FinalVersion_V2.dgn
+              MyFileWithAReallyLongNameThatWillBeSplitIntoMultipleLinesAfterSoLong_FinalVersion_V2.svg
```

Comment:
Is the change on line 704 (`.dgn` -> `.svg`) because the icon on line 702 is of the type vector/svg?